tests/containers/rootless_docker.pm

```diff
@@ -33,7 +33,8 @@ sub run {
 
     my $docker = containers::docker->new();
 
-    install_packages('docker-rootless-extras');
+    my $pkg_name = check_var("CONTAINERS_DOCKER_FLAVOUR", "stable") ? "docker-stable" : "docker";
```

Comment:
Should `stable` really be the default here?

Reply:
It's not a default. Default is undefined and that defaults to `docker`.

Reply:
Right, I misread it as `get_var`